less/index.less

```diff
@@ -49,11 +75,180 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+html {
+	box-sizing: border-box;
+	font-family: @font-stack;
+}
+
 * {
-    box-sizing: border-box;
+	box-sizing: inherit;
+}
+
+h1, h2, h3, h4, h5 {
+	color: @main-color;
+    font-size: 1.8rem;
+    margin-bottom: 1.5rem;
+    font-family: @font-stack;
+}
+
+p {
+    line-height: 1.4;
+}
+
+.container {
+    width: 800px;
+	margin: 0 auto;
+
+	@media @mobile {
+		width: 90%;
+		margin: 0 auto;
+	}
+}
+
+/*header styles*/
+
+header {
+	width: 100%;
+	.flex1;
+	justify-content: flex-end;
+
+	nav  {
+		width: 30%;
+		.flex1;
+		justify-content: space-between;
+		margin: 2rem 0;
+
+		@media @mobile {
+			width: 100%;
+			.flex1;
+		}
+
+	}
+
+	nav a {
+		color: black;
+		text-decoration: none;
+	}
+	
+	nav a:hover{
+		text-decoration: underline;
+		color: gray;
+	}
+
 }
 
-html, body {
-    height: 100%;
+
+
+/* content styles*/
+
+/*top content*/
+.top-content {
+	background: #e9ecef;
+	border: 1px;
+	border-radius: 1%;
+	padding: 10% 5% .5% 5%;
+	width: 100%;
+
+
+	.arizona-img {
+		width: 50%;
+
+		@media @mobile {
+			width: 100%;
+		}
+	}
+	
+	.text-container {
+		margin-top: 1rem;
+	}
+	
+	.text-container2 {
+		margin: 2rem 0;
+		padding: 2rem 0;
+		border-top: 1px solid lightgrey;
+	}
+	
+	button {
+		background: @accent-color;
+		color: white;
+		margin-top: 2rem;
+		padding: 1rem 4%;
+		font-size: 1.7rem;
+		border: 1px solid black;
+		border-radius: 5px;
+		cursor: pointer;
+
+		@media @mobile {
+			width: 100%;
+			margin: 1rem auto;
+			padding: 1rem;
+			}
+
+	}
+	
+		button:hover {
+			background: black;
+			color: white;
+		}
```

Comment:
just personal preference, just to be safe with cascading, i would put your button:hover over the media query, that way the hover is rendered first, which makes more sense to do since you want the hover working on every media query.
then again, its totally up to you.